### ISSUE TITLE
Erroneous handling of quadrupole TILT in presence of K1S

### ISSUE THREAD

**Issue by chernals** [MethodicalAcceleratorDesign/MAD-X#840]
The documentation states the following:

> Note also that K1 or K1S can be considered as the normal or skew components of the magnet on the bench, while the TILT attribute can be considered as a tilt alignment _error_ in the machine. [...]

(note: I would rephrase _error_ as _value_, as it could be a purposedly rotated quadrupole)

However, the code works like this (see `tttquad` in `ttrun.f90`):

```
 tilt = g_elpar(q_tilt)

  [...]
  k1  = g_elpar(q_k1)
  k1s = g_elpar(q_k1s)

 [...]

  if (k1s.ne.zero) then
     tilt = -atan2(k1s, k1)/two ! + tilt
     k1 = sqrt(k1**2 + k1s**2)
  else
     tilt = zero
  endif
```

So apparently the _TILT_ is *not* added to the effective tilt induced by K1 and K1S.

I also see this in the documentation:

> Please note that contrary to MAD-8 one has to specify the desired TILT angle, otherwise it is taken as 0 rad. This was needed [...]

For sure I would say that the documentation contradicts itself. I would also suggest to restore the behavior of MAD-8.

**Comment by tpersson** [comment]
The tilt is added before and after the element. I made a very quick test and it behaves as I expect it. The tilt you are referring to is the relative strength of the k1 and k1s. All this could be done in one place but this is not how it is implemented. 

I agree that the documentation could be improved and I will try fix it.

**Comment by tpersson** [comment]
I just saw that there is an additional tilt in the cod. I will check if this is correct..

**Comment by chernals** [comment]
@tpersson If I understand correctly your comment here and in #841 the `tilt` attributes implies a rotation that is done outside the tracking routine of the element itself (and that's why I didn't see it apparently). Is that statement correct?

If so, then the documentation could be improved, as, the way it is now, it would imply that only one rotation is done (either tilt or atan2(k1, k1s)/2).
